spark/src/main/java/org/opensearch/sql/spark/dispatcher/SparkQueryDispatcher.java

```diff
@@ -79,6 +79,9 @@ private AsyncQueryHandler getQueryHandlerForFlintExtensionQuery(
       return queryHandlerFactory.getIndexDMLHandler();
     } else if (isEligibleForStreamingQuery(indexQueryDetails)) {
       return queryHandlerFactory.getStreamingQueryHandler();
+    } else if (IndexQueryActionType.CREATE.equals(indexQueryDetails.getIndexQueryActionType())) {
+      // create should be handled by batch handler
+      return queryHandlerFactory.getBatchQueryHandler();
```

Comment:
Is this handler being used anywhere today? sessionManaged.isEnabled is always true so even getDefaultAsyncQueryHandler would always return InteractiveQueryHandler. I hope there won't be any regression. 

Also RefreshQueryHandler is also using FlintJob Right?

Reply:
> Is this handler being used anywhere today?

I believe no

> RefreshQueryHandler is also using FlintJob Right?

yes. RefreshQueryHandler extends BatchQueryHandler